common/src/main/java/com/wynntils/models/stopwatch/StopwatchModel.java

```diff
@@ -0,0 +1,63 @@
+package com.wynntils.models.stopwatch;
+
+import com.wynntils.core.components.Model;
+
+import java.util.List;
+
+public class StopwatchModel extends Model {
+
+    private long startTimeMillis = -1;
+    private long currentTimeMillis = -1;
+    private boolean running = false;
+
+    public StopwatchModel() {
+        super(List.of());
+    }
+
+    public int getHours() {
+        update();
```

Comment:
If you are going to call cTM() at each getter, there is no need to store the value in a field..?